Development environment setup.md

```diff
@@ -1,10 +1,14 @@
 ## Install basic tools
 
-- [Latest Oracle JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
 - [Android Studio](https://developer.android.com/studio/index.html)
 - Git—`brew install git` ([brew docs](http://brew.sh/))
 - Zeplin—`brew cask install zeplin` ([brew cask docs](https://caskroom.github.io/))
 
+## Setup java JDK
+
+- On welcome screen go to Configure -> Default project structure... and change JDK location to Embedded JDK
```

Comment:
@jmucnjak Can you put one sentence like in the description of the PR. So it is clear that JDK is now embedded in the AS and if we have some other environment that uses latest (or any other JDK) to ensure AS is using the embedded one.